api/src/main/resources/workbench.yaml

```diff
@@ -1335,6 +1338,7 @@ paths:
     post:
       tags:
         - workspaces
+        - compute
```

Comment:
I don't know whether this should be tagged. This is different because lockout should be applied to the destination (found in the request payload), and is unrelated to the source workspace in the URL. Therefore any kind of interceptor-based approach will not really be able to use this tag to apply any standard URL-based filtering.

Reply:
I'm also nervous because our API structure wasn't originally designed to be partitioned along these lines. I'd almost rather see a v2 API that separates billable and non-billable endpoints by path. I imagine that's a nonstarter for the first launch though.

Reply:
Which workspace gets billed after a copy operation? I did some googling but was unable to find a clear answer.

I'll ask Karthik what he thinks the right product behavior is. From a technical perspective, we can move the billing check into the controller method if we want to check the destination workspace's status.

Reply:
The billing costs involved here are purely coming from GCS. There might be a small egress charge to the source workspace during a copy, but if so I think it's negligible enough to ignore. If we copy a file into a destination workspace bucket, that workspace will be billed monthly for any storage costs.

The goal with billing lockout would be to prevent increases in workspace cost, namely due to the user creating a cluster, or (less significantly) adding GCS objects to the workspace (increasing the storage cost per month).

Reply:
If it makes it easier (/ possible) to use this approach for billing lockout, given our time constraints, I would be ok with disabling copying the notebook to another workspace - but ultimately the behavior of the app is a product decision.

Reply:
What would be the advantage of copying the notebook in this situation? The other workspace could still have a billing acct/free credits?

Reply:
Correct.